docs/Using-Fleet/MDM-commands.md

```diff
@@ -71,5 +71,18 @@ Fleet UI:
 
 2. Look at the on-screen information.
 
+## List of recent commands
+
+You can view the list of recently executed commands using "fleetctl":
+
+1. Run `fleetctl get mdm-commands`
+2. View the list of recently executed commands, most recent first, along with the timestamp, targeted hostname, command type, execution status and command ID.
+
+The command ID can be used to view command results as documented in [Step 4 of the previous section](#step-4-view-the-commands-results). The possible status values are:
+* Pending: the command has yet to be executed on the host.
+* Acknowledged: the command was executed successfully.
+* Error: execution of the command failed.
```

Comment:
```suggestion
* Error: the host responded with "Error" status via the MDM protocol: an error occurred. Run the `fleetctl get mdm-command-results --id=<insert-command-id` to view the error.
```

I think we should be explicit here too. We're taking the status from the MDM protocol right?

Reply:
Yes, that's correct.